configure_project/versions.tf

```diff
@@ -0,0 +1,9 @@
+terraform {
+  required_version = ">= 1.5.0"
```

Comment:
Just set required_version = ">= 1.5.0, <1.7.0" . This is to prevent from using hashicorp tf 1.7 and above as the DA won't be tested with these versions in pipeline.

Reply:
This applies across the module

Reply:
done